client/src/Data/Api.elm

```diff
@@ -2,7 +2,7 @@ module Data.Api exposing
     ( Endpoint(..), get, post, put, patch, delete
```

Comment:
Instead of rejecting the checkout, can we display a warning after checking out success instead?

Something like "Checkout succeed ... Your address is being verified, we will contact you further for detail .."

I think rejecting the checkout all together can be a problem, since if my address is not in postgrid, I cannot buy anything at all.

[Screencast from 08-29-2025 07:26:02 AM.webm](https://github.com/user-attachments/assets/132c77b2-80fd-4197-add9-d10234db2609)

Reply:
Discussion continued in thread: https://github.com/Southern-Exposure-Seed-Exchange/southernexposure.com/pull/90#discussion_r2308860126

Reply:
> since if my address is not in postgrid, I cannot buy anything at all.

This implies that the address is not known to USPS, and thus, it won't be possible to send a delivery